fe2-android/app/src/test/java/com/github/dedis/popstellar/repository/LAORepositoryTest.java

```diff
@@ -1,4 +1,4 @@
-package com.github.dedis.popstellar.repository;
+/*package com.github.dedis.popstellar.repository;
```

Comment:
I assume here you wanted to `@Ignore` the tests rather than comment out the whole file ? 
Is the issue at compile-time or with the test execution ?

Reply:
It is compliation issue, and don't worry, I do not plan to push this to master. This is more of a Work in Progress to push the code and let the CI run once.

Reply:
As the tests were only focused on the protocol, which is now handled by LAONetworkManager, this test suite is obsolete.